README.md

```diff
@@ -35,6 +35,13 @@ The documentation is available at
 - Make sure you have [sbt](https://www.scala-sbt.org/) installed.
 - [Graphviz](https://graphviz.gitlab.io/download/) is needed for the scaladoc generation build task, which is part of the release.
 
+### h2spec
```

Comment:
Wouldn't this section make more sense in `CONTRIBUTING.mg`, as it's a detail only relevant to people building/testing the project and not users.

Reply:
This affects anyone who runs from tests on a Mac Silicon. Not necessarily a contributor.